### PR TITLE
Remove threads buttons

### PR DESCRIPTION
**Pull request by jdauphant**
Cf. issue https://github.com/tchapgouv/tchap-web-v4/issues/462

Remove thread buttons

1. First occurrence
- before:
Screenshot 2023-03-22 at 11 26 54

-after:
Screenshot 2023-03-22 at 11 19 11

2. Second occurrence
- before:
Screenshot 2023-03-22 at 11 27 09

- after:
Screenshot 2023-03-22 at 11 19 03

### CODE REVIEW

File: patches/remove-thread-buttons/matrix-react-sdk+3.68.0.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+diff --git a/node_modules/matrix-react-sdk/src/components/views/context_menus/MessageContextMenu.tsx b/node_modules/matrix-react-sdk/src/components/views/context_menus/MessageContextMenu.tsx
+index ad56409..fa0407d 100644
+--- a/node_modules/matrix-react-sdk/src/components/views/context_menus/MessageContextMenu.tsx
++++ b/node_modules/matrix-react-sdk/src/components/views/context_menus/MessageContextMenu.tsx
+@@ -678,7 +678,9 @@ export default class MessageContextMenu extends React.Component<IProps, IState>
+                 <IconizedContextMenuOptionList>
+                     {reactButton}
+                     {replyButton}
++                    {/* TCHAP : Remove Reply In Thread Button
```
Comment (odelcroi) [inline patches/remove-thread-buttons/matrix-react-sdk+3.68.0.patch]: we use normally  ':tchap:' and ':tchap: end' as tags

**Comment by odelcroi** [comment]
https://github.com/tchapgouv/tchap-web-v4/issues/472

**Comment by odelcroi** [comment]
https://github.com/tchapgouv/tchap-web-v4/issues/462

File: patches/remove-thread-buttons/matrix-react-sdk+3.68.0.patch [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+             );
+diff --git a/node_modules/matrix-react-sdk/src/components/views/messages/MessageActionBar.tsx b/node_modules/matrix-react-sdk/src/components/views/messages/MessageActionBar.tsx
+index 1a7d773..4e8f18f 100644
+--- a/node_modules/matrix-react-sdk/src/components/views/messages/MessageActionBar.tsx
++++ b/node_modules/matrix-react-sdk/src/components/views/messages/MessageActionBar.tsx
+@@ -367,18 +367,21 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
+     private readonly forbiddenThreadHeadMsgType = [MsgType.KeyVerificationRequest];
+ 
+     private get showReplyInThreadAction(): boolean {
+-        const inNotThreadTimeline = this.context.timelineRenderingType !== TimelineRenderingType.Thread;
```
Comment (jdauphant) [inline patches/remove-thread-buttons/matrix-react-sdk+3.68.0.patch:23]: @aulamber c'est possible d'avoir une capture d'écran de ce changement ?

**Review by odelcroi — APPROVED** [review]

**Pull request merged**